en/mkdocs.yml

```diff
@@ -48,15 +48,43 @@ nav:
   - 'Home': index.md
   - 'Quick Starts':
       - 'Overview': quick-starts/overview.md
-      - 'Mobile':
-          - 'Android': quick-starts/android.md
-          - 'iOS': quick-starts/ios.md
-      - 'Frontend':
-          - 'Angular': quick-starts/angular.md
-          - 'React': quick-starts/react.md
-      - 'Backend':
-          - 'Java': quick-starts/java.md
-          - 'NodeJs': quick-starts/nodejs.md
+      - 'Choose an SDK':
+        - 'Android': quick-starts/android.md
+        - 'React': quick-starts/react.md
+        - 'Java': quick-starts/java.md
+      - 'Choose a Sample Type':
+        - 'Android': quick-starts/android-sample.md
+        - 'React': quick-starts/react-sample.md
+        - 'Java': quick-starts/java-sample.md
+      - 'Authentication':
+        - 'OpenID Connect': quick-starts/regular-webapp-oidc-sample.md
+        - 'SAML': quick-starts/regular-webapp-saml-sample.md
+      - 'Single Sign On':
+        - 'OAuth/OIDC Apps': quick-starts/sso-for-oidc-apps.md
+        - 'SAML Apps': quick-starts/sso-for-saml-apps.md
+        - 'Different Authentication Protocols': quick-starts/cross-protocol-sso.md
+      - 'Multi-Factor Authentication': quick-starts/mfa-sample.md
+      - 'Adaptive Authentication':
+        - 'Role-Based': quick-starts/role-based-adaptive-auth.md
+        - 'User Age-Based': quick-starts/age-based-adaptive-auth.md
+        - 'Tenant-Based': quick-starts/tenant-based-adaptive-auth.md
+        - 'User Store-Based': quick-starts/user-store-based-adaptive-auth.md
+        - 'IP-Based': quick-starts/ip-based-adaptive-auth.md
+        - 'Device-Based': quick-starts/device-based-adaptive-auth.md
+        - 'Login Attempts-Based': quick-starts/login-attempts-based-adaptive-auth.md
+        - 'ACR-Based': quick-starts/acr-based-adaptive-auth.md
+        - 'Using Function Library': quick-starts/adaptive-auth-with-function-lib-sample.md
+        - 'Limiting Active User Sessions': quick-starts/limiting-active-sessions-adaptive-auth.md
+      - 'Access Delegation':
+        - 'Authorization Code Grant': quick-starts/auth-code-playground.md
+        - 'Implicit Grant': quick-starts/implicit-playground.md
+        - 'Client Credentials Grant': quick-starts/client-credentials-playground.md
+        - 'Password Grant': quick-starts/password-playground.md
+        - 'Microprofile JWT': quick-starts/microprofile-jwt-sample.md
+        - 'SAML2 Bearer Assertion Profile': quick-starts/saml2-bearer-assertion-profile.md
+        - 'Encrypt ID Tokens': quick-starts/oidc-token-encryption-sample.md
+        - 'OpenID Connect Back-Channel Logout': quick-starts/oidc-backchannel-logout-sample.md
+        - 'Passing OIDC Parameters as a Request Object': quick-starts/request-object.md
```

Comment:
Since we have moved all these into the quick-starts folder from the samples folder, if we merge this now like this, there will be a lot of broken links from the guides section where we have still given the link path to the samples folder...